e2e_tests/integration/rationale.feature

```diff
@@ -180,7 +180,7 @@ Feature: Rationale for a Recall decision
     # MRD-1262: AC0/AC1
     And PO requests an SPO to countersign
     And SPO has visited the countersigning link
-    And SPO has recorded rationale with <SPODecision> decision
```

Comment:
This step was written to cater to both RECALL & NO RECALL decisions and before SPO has countersigned, if the intention of this step is to be called SPO countersignature, then there's another step called, which need to be changed to only allow RECALL

```ghekin
Then SPO is able to record rationale with RECALL decision
```
```javascript
Then('SPO is able to record rationale with {managersDecision} decision', function (decision: string) {
  recordSpoDecision.call(this, decision)
})
```